index.js

```diff
@@ -41,28 +41,85 @@ var makiAvailable = fs.readdirSync(makiRenders)
 
 module.exports = getMarker;
 
-function convertFromHexSrgb(value) {
-    // Parse hex to a float in [0..1]
-    const f = parseInt(value, 16) / 255.0;
-    // Transform 0-1 sRGB float to the vector expected by the relative luminance formula.
+// White
+const lightColor = 'ffffff';
+// Base10
+const darkColor = '202122';
+
+/**
+ * @param {string} color 6-digit hex color like '7e7e7e'
+ * @returns {[integer, integer, integer]} sRGB color with 0-255 integer components.
+ */
+function hexToSrgb(color) {
+    return color
+        .match(/../g)
+        .map((value) => parseInt(value, 16));
+}
+
+/**
+ * Improve on blend.parseTintString, to pass through the lightness component
+ * without the bad contrast math.
+ *
+ * @param {string} color 6-digit hex color
+ * @returns {object} HSL with [0-1] float components.  Each term is a redundant
```

Comment:
I don't think the more detailed type improves readability, though?

@returns seems to be the canonical name: https://jsdoc.app/tags-returns.html and it's only the wikimedia eslint plugin that enforces @return.  On that topic, maybe we should turn on a linter, allow ES6, and rewrite `var` for example?

Reply:
Our canon is what the Wikimedia linter says. And we are in a Wikimedia codebase here, aren't we? But maybe this is an exception because it's a fork from something outside of the org. Let's not touch code style details for now.

Reply:
+1 that we've restyled repos in the past.  Also +1 that it's a bigger job and best done by eslint --fix